spec/unit/coinbase/wallet_spec.rb

```diff
@@ -21,10 +21,12 @@
   let(:second_address_model) do
     build(:address_model, network_id, :with_seed, seed: seed, wallet_id: wallet_id, index: 1)
   end
+  let(:webhook_model) { build(:webhook_model, :wallet_activity) }
```

Comment:
If this is only used in the `#create_webhook` test, please define it in that test.

The `let` should generally be defined in the most narrow scope possible (e.g. the top-level `lets` are referenced by multiple test cases or in a top-level `before` block_

Reply:
could it be Eth?

Reply:
@chaoyaji-cb can this be define in your test alone?

Reply:
done!